src/utils/expression-runner.ts

```diff
@@ -168,6 +168,27 @@ export function evaluateExpression(
       : null;
   }
 
+  function validateAge(questionId) {
```

Comment:
@melkam-mekonnen can you suggest more descriptive naming for this function? The function above doesn't validate a person's age as it sounds from the function's name. Does it get a difference between a `field.dateValue` and `patient. birthDate`?